perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -814,34 +800,46 @@ sub report_edit : Path('report_edit') : Args(1) {
         : [],
         print_report => 1,
     );
+}
 
-    if (my $rotate_photo_param = $self->_get_rotate_photo_param($c)) {
-        $self->rotate_photo($c, $problem, @$rotate_photo_param);
-        if ( $c->cobrand->moniker eq 'zurich' ) {
-            # Clicking the photo rotation buttons should do nothing
-            # except for rotating the photo, so return the user
-            # to the report screen now.
-            $c->res->redirect( $c->uri_for( 'report_edit', $problem->id ) );
-            return;
-        } else {
-            return 1;
-        }
-    }
+sub report_edit : Path('report_edit') : Args(1) {
+    my ( $self, $c, $id ) = @_;
 
-    $c->forward('categories_for_point');
+    my $problem = $c->cobrand->problems->search( { id => $id } )->first;
```

Comment:
I assume this has to be `search` and not a `find` as the Resultset returned by `$c->cobrand->problems` prevents that, and `id` has to be unique so this will never return more than one row?

Reply:
`find` would be fine here I think (though yes, I think search on primary key is basically equivalent), that's just the existing code (says it's part of the diff as it got moved around, `git diff --histogram` is a bit nicer on this commit for some reason).